CleverPush/Source/CleverPushInstance.m

```diff
@@ -1665,9 +1666,28 @@ - (void)didRegisterForRemoteNotifications:(UIApplication*)app deviceToken:(NSDat
 #pragma mark - Generalised Api call.
 - (void)enqueueRequest:(NSURLRequest*)request onSuccess:(CPResultSuccessBlock)successBlock onFailure:(CPFailureBlock)failureBlock {
     [CPLog info:@"[HTTP] -> %@ %@", [request HTTPMethod], [request URL].absoluteString];
+    NSURLRequest *modifiedRequest;
+    if ([CleverPush getAuthorizerToken] != nil && ![[CleverPush getAuthorizerToken] isKindOfClass:[NSNull class]] && ![[CleverPush getAuthorizerToken] isEqualToString:@""]) {
+        NSMutableURLRequest *urlRequest = [request mutableCopy];
+        NSError *error;
+        NSMutableDictionary *requestParameters = [[NSJSONSerialization JSONObjectWithData:[urlRequest HTTPBody] options:0 error:&error] mutableCopy];
```

Comment:
There is no way around this JSON parsing + JSON stringifying @cpandya25 ? It would be better if we can prevent this.

Reply:
https://ibb.co/4KfH0Hn

Reply:
Hello @gbhrdt ,

https://ibb.co/vwmZXDr

The above method is used for getting the existing parameters, and after that, we need to add the new token parameter.